nuxeo-opencmis-impl/src/main/java/org/nuxeo/ecm/core/opencmis/impl/server/NuxeoCmisConstants.java

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
Second line must be `(C) Copyright 2017 Nuxeo SA (http://nuxeo.com/) and others.` then an empty line, then the Apache license. Please use the exact license wording and formatting from https://doc.nuxeo.com/corg/copyright-and-license-headers/